notify/src/kqueue.rs

```diff
@@ -314,7 +316,7 @@ impl EventLoop {
             .add_filename(&path, event_filter, filter_flags)
             .map_err(|e| Error::io(e).add_path(path.clone()))?;
         self.watches.insert(path, is_recursive);
-        self.kqueue.watch()?;
```

Comment:
Can we add a comment to `add_single_watch` that it requires calling `kqueue.watch` by the caller ?

Reply:
Done in https://github.com/notify-rs/notify/pull/454/commits/f6da0db4d5bb236b7f4f71e093ec21aa6cb0f0ee